lib/icinga/clusterevents.cpp

```diff
@@ -1115,12 +1115,7 @@ Value ClusterEvents::UpdateExecutionsAPIHandler(const MessageOrigin::Ptr& origin
 	updateMessage->Set("method", "event::UpdateExecutions");
 	updateMessage->Set("params", params);
 
-	Zone::Ptr localZone = Zone::GetLocalZone();
-	Zone::Ptr parentZone = localZone->GetParent();
-
-	if (localZone != parentZone) {
-		listener->RelayMessage(origin, parentZone, updateMessage, true);
-	}
+	listener->RelayMessage(origin, Zone::GetLocalZone(), updateMessage, true);
```

Comment:
And this one covers the parent zone?

Reply:
Yes, from my tests, for example, if an ExecuteCommand request is sent directly to the agent with `command_enpoin = agent` the result is sent to the satellite and also to the master